backend/chain.py

```diff
@@ -236,39 +236,42 @@ def cohere_response_synthesizer(input: dict) -> RunnableSequence:
     )
 
 
-llm = ChatOpenAI(
-    model="gpt-3.5-turbo-0125",
+gpt_3_5 = ChatOpenAI(model="gpt-3.5-turbo-0125", temperature=0, streaming=True)
```

Comment:
Modify doc needs to be updated if you're removing the `configurable_alternatives` usage
https://github.com/langchain-ai/chat-langchain/blob/master/MODIFY.md#llm

Reply:
not removing it

Reply:
see L264